### PR TITLE
Random Acts of Improvement

### PR DESCRIPTION
**Pull request by dbrabera**

### CODE REVIEW

File: zendesk/doc.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//        client, err := zendesk.NewClient("domain", "username", "password")
+//        if err != nil {
+//            log.Fatal(err)
+//        }
+//        ticket, err := client.ShowTicket(1)
+//        if err != nil {
+//            log.Fatal(err)
+//        }
+//        log.Printf("Requester ID is: %d", *ticket.RequesterID)
+//    }
```
Comment (ssimono) [inline zendesk/doc.go]: Do you have a strategy to make sure this example (and its README equivalent) stays up to date and always compiles?

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,3 @@
-include .env
```
Comment (ssimono) [inline Makefile]: This was made intentionally. Manually exporting some variables (and finding their values) each time you want to hack into the library is a real pain. This things makes it easy for developpers

File: zendesk/doc.go (hunk 1 above)
Comment (dbrabera) [inline zendesk/doc.go]: No. I don't think anything like it's in common use or exist. I think it's very useful to have code samples in the documentation, but as with any piece of documentation, that's the risk to pay.

File: Makefile (hunk 2 above)
Comment (dbrabera) [inline Makefile]: I restored it, but you don't have to manually do it either. You can just `source` your env file before starting to develop.

**Pull request merged**